docs/articles/index.md

```diff
@@ -0,0 +1,136 @@
+
+---
+
+## A
+[Access Control](access-control.md)  
+[ACLs](access-control.md)  
+[API](api.md)  
+[Application Groups](application-groups.md)  
+[Applications](applications.md)  
+[Artifacts](artifacts.md)  
+[Authentication](access-control.md)  
+[Authorization](access-control.md)  
+
+## B
+[Backoff Strategy](backoff-strategy.md)  
+[Bridge Networking](networking.md#bridge-networking)
+
+## C
+[CLI](cli.md)  
+[Cluster Maintenance](cluster-maintenance.md)  
+[Command Line Arguments](command-line-arguments.md)  
+[Configuration](configuration.md)  
+[Constraints](constraints.md)  
+[Containers](containers.md)  
+[Custom Executor](executors.md)  
+
+## D
+[Debugging](debugging.md)  
+[Dependencies](dependencies.md)  
+[Deployments](deployments.md)  
+[Docker](docker.md)  
+[Download](download.md)  
+[Downscaling](downscaling.md)  
+
+## E
+[Endpoints](endpoints.md)  
+[Env Var Based Secrets](secrets.md)  
+[Environment Variables](environment-variables.md)  
+[Ephemeral Volumes](ephemeral-volumes.md)  
+[Event-bus](event-bus.md)  
+[Events](events.md)  
+[Executors](executors.md)  
+[Exponential Backoff](backoff-strategy.md)  
+[External Volumes](external-volumes.md)  
+[Features](features.md)  
+
+## F
+[File Based Secrets](secrets.md)  
+
+## H
+[Health Checks](health-checks.md)  
+[Host Networking](host-networking.md)  
+[HTTP API](api.md)  
+
+## I
+[Installation](installation.md)  
+[Instance Lifecycle](instance-lifecycle.md)  
+[Instances](instances.md)  
+
+## K
+[Kill Grace Period](kill-grace-period.md)  
+[Kill Selection](kill-selection.md)  
+
+## L
+[Labels](labels.md)  
+[Limitations](limitations.md)  
+[Load Balancing](load-balancing.md)  
+[Local Volumes](local-volumes.md)  
+
+## M
+[Marathon Health Checks](marathon-health-checks.md)  
+[Mesos Compatibility](mesos-compatibility.md)  
+[Mesos](mesos.md)  
+[Mesos Executor](executors.md)  
+[Mesos Health Checks](mesos-health-checks.md)  
+[Metrics](metrics.md)  
+[Migration](migration.md)  
+[Mounts](mounts.md)  
+
+## N
+[Networking](networking.md)  
+
+## P
+[Packages](packages.md)  
+[Parameters](parameters.md)  
+[Persistent Volumes](persistent-volumes.md)  
+[Plugins](plugins.md)  
+[Pods](pods.md)  
+[Port Definitions](networking.md#port-definitions)
+[Port Mappings](networking.md#port-mappings)
+
+## R
+[Rate Limiting](backoff-strategy.md)  
+[Readiness Checks](readiness-checks.md)  
+[Recipes](recipes.md)  
+[Recommendations](recommendations.md)  
+[Resource Requirements](resource-requirements.md)  
+[Roles](roles.md)  
+
+## S
+[Scaling](scaling.md)  
+[Secrets](secrets.md)  
+[Security](access-control.md)  
+[Service Discovery](service-discovery.md)  
+[Services](services.md)  
+[Sharding](sharding.md)  
+[SSL](ssl.md)  
+[Stateful Applications](stateful-applications.md)  
```

Comment:
`Stateful Apps`?

Reply:
See above – Apps is an abbrev. so I was thinking we should go for Application.